src/main/groovy/com/vanniktech/maven/publish/MavenPublishPlugin.groovy

```diff
@@ -39,10 +39,27 @@ class MavenPublishPlugin extends BaseMavenPublishPlugin {
       options.linksOffline "https://developer.android.com/reference", "${project.android.sdkDirectory}/docs/reference"
     }
 
-    project.tasks.create("androidJavadocsJar", Jar.class) {
+    def androidJavadocsJar = project.tasks.create("androidJavadocsJar", Jar.class) {
       classifier = 'javadoc'
-      from project.androidJavadocs.destinationDir
-    }.dependsOn("androidJavadocs")
+    }
+
+    if (plugins.hasPlugin('kotlin-android')) {
+      def dokkaOutput = "${project.docsDir}/dokka"
+      project.plugins.apply('org.jetbrains.dokka-android')
+      project.dokka {
+        outputFormat 'javadoc'
```

Comment:
Any reason this was chosen over `html`?

Reply:
I was also confused - https://github.com/vanniktech/gradle-maven-publish-plugin/pull/37#issuecomment-454016415